src/managers/config.py

```diff
@@ -113,7 +113,7 @@ def __init__(
         self.node_port = node_port
 
     @property
-    def scope(self) -> Scope:
+    def scope(self) -> str:
```

Comment:
**todo:** This is a `Scope`?

Reply:
`pyright` is complaining about Scope :(

Reply:
Strange, it never used to. What happens if we change the type of Scope itself to `Scope(str, Enum)`?

Reply:
Added a cast to help pyright :)